java/cdk/src/main/java/sleeper/cdk/stack/AthenaStack.java

```diff
@@ -156,7 +154,7 @@ private IFunction createConnector(String className, String instanceId, int logRe
                 .memorySize(memory)
                 .timeout(Duration.seconds(timeout))
                 .runtime(Runtime.JAVA_11)
-                .logRetention(Utils.getRetentionDays(logRetentionDays))
+                .logGroup(createLogGroupWithRetentionDays(this, simpleClassName + "AthenaCompositeHandlerLogGroup", logRetentionDays))
```

Comment:
If we include LogGroup in the ID for the CDK resource, I think that will also appear in the names of the log groups by default. That seems a bit messy to view in a list of log groups. Maybe we should set the log group names directly, separately from the CDK ID?